packages/mds-logger/index.ts

```diff
@@ -14,59 +14,30 @@
  * limitations under the License.
  */
 
-import httpContext from 'express-http-context'
-import pino, { Logger } from 'pino'
 import { inspect } from 'util'
-
+import { LogLevel } from './@types'
+import { debugLog, logger } from './loggers'
+import { redact } from './redaction'
 // Verify that defaultOptions is available (not available on non-nodejs platforms)
 if (inspect?.defaultOptions) {
   // Set the inspector depth to infinity. To the moooooooon 🚀 🌕
   inspect.defaultOptions.depth = null
 }
 
-const logger: Pick<Logger, 'info' | 'warn' | 'error' | 'debug'> = pino(pino.destination({ sync: false }))
-type LogLevel = keyof typeof logger
-
-const redact = (arg: string | Record<string, unknown> | Error | undefined) => {
-  if (arg === undefined) return {}
-  const res = JSON.stringify(arg instanceof Error ? { error: arg.toString() } : arg, (k, v) =>
-    ['lat', 'lng'].includes(k) ? '[REDACTED]' : v
-  )
-
-  return JSON.parse(res)
-}
-
 const log =
   (level: LogLevel) =>
-  (
-    message: string,
-    data?: Record<string, unknown> | Error
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  ): { log_message?: string; log_data?: any } => {
+  (message: string, data?: Record<string, unknown> | Error): void => {
     if (process.env.QUIET === 'true') {
-      return {}
+      return
```

Comment:
Now that this is a `void` function, you could just wrap the entire body in:
```ts
if (process.env.QUIET !== 'true') {
  ...
}
```

Reply:
Resolved in b199007